src/Xamarin.Android.Build.Tasks/Linker/MonoDroid.Tuner/StripEmbeddedLibraries.cs

```diff
@@ -18,7 +18,9 @@ protected override void ProcessAssembly (AssemblyDefinition assembly)
 				return;
 
 			var fileName = assembly.Name.Name + ".dll";
-			if (MonoAndroidHelper.IsFrameworkAssembly (fileName) && !MonoAndroidHelper.FrameworkEmbeddedJarLookupTargets.Contains (fileName))
+			if (MonoAndroidHelper.IsFrameworkAssembly (fileName) &&
+					!MonoAndroidHelper.FrameworkEmbeddedJarLookupTargets.Contains (fileName) &&
+					!MonoAndroidHelper.FrameworkEmbeddedNativeLibraryAssemblies.Contains (fileName))
```

Comment:
I don't understand this change.

`Mono.Posix.dll` and `Mono.Data.Sqlite.dll` will both contain native libraries (within the `.zip` resource).  *Surely* we want to *continue* to remove the native libraries from these assemblies, don't we?